cloudify_agent/installer/config/configuration.py

```diff
@@ -211,6 +211,11 @@ def reinstallation_attributes(cloudify_agent):
     _add_installation_defaults(cloudify_agent)
 
 
+def _get_manager_file_server_url(cloudify_agent):
+    ip = cloudify_agent.get('manager_ip')
+    return 'http://{}:53229'.format(ip)
```

Comment:
Maybe it's better to alter the get_manager_file_server_url() to optionally receive an ip. if the ip is none, it will revert to the current usage. either way, i think it's better to write the function in the following way:
```
'{0}://{1}:{2}'.format(get_manager_file_server_protocol(), ip, get_manager_file_server_port())
```